api/api.js

```diff
@@ -0,0 +1,18 @@
+import fetch from 'isomorphic-fetch';
+
+
+const api = {
+    urlApi: 'https://pereiratechtalks.com',
+    PTTpost: {
+        async posts(){
+            const queryParams = '?limit=9&filter=author%3Apereirajs&client_id=ghost-frontend&client_secret=7709d8d5ac20'
+            const response = await fetch(`https://pereiratechtalks.com/ghost/api/v0.1/posts/${queryParams}`);
```

Comment:
The built URL looks like a constant, may we take it away from the function?

Reply:
Yes, we do. Some refactory is missing here. We will do that